packages/jest-cli/src/jest.js

```diff
@@ -347,6 +353,10 @@ const runCLI = (
               pipe.write('\n');
               writeCurrentPattern();
               break;
+            case KEYS.I:
+              argv.watchInterrupt = true;
+              console.log('\n' + chalk.bold('Watch Interrupted...'));
```

Comment:
it should be `pipe.write(chalk.bold('Test run interrupted'))`. Let's get rid of the `...` here too.